diff --git a/docker-compose.yml b/docker-compose.yml
index 5927d39..6939594 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -14,18 +14,26 @@ services:
     ports:
       - 8000:8000
     command: /api
+    networks:
+      - front-back
 
 # WEB
   client:
     build: ./packages/web
     container_name: web
     expose:
-      - 80 443
+      - 443
     ports:
       - 80:80
       - 443:443
     links:
       - "server"
     command: nginx -g 'daemon off';
-    volumes:
+    volumes: 
       - ./data/certs:/etc/nginx/certs
+    networks:
+      - front-back
+
+networks:
+  front-back:
+    driver: bridge
diff --git a/packages/web/nginx.conf b/packages/web/nginx.conf
index 695d625..174f683 100644
--- a/packages/web/nginx.conf
+++ b/packages/web/nginx.conf
@@ -10,4 +10,9 @@ server {
     include /etc/nginx/mime.types;
     try_files $uri $uri/ /index.html;
   }
+
+  location /api {
+    proxy_pass http://api:8000;
+  }
+
 }
diff --git a/packages/web/src/Global.js b/packages/web/src/Global.js
index 3478ed6..efd12d1 100644
--- a/packages/web/src/Global.js
+++ b/packages/web/src/Global.js
@@ -8,7 +8,7 @@ const env = process.env.NODE_ENV;
 let url = null;
 
 // eslint-disable-next-line no-undef
-if (process.env.REACT_APP_DEV_URL) {
+if (process.env.REACT_APP_DEV_URL != null) {
     // DEV ENVIRONMENT
     // eslint-disable-next-line no-undef
     url = process.env.REACT_APP_DEV_URL;
